carbonj.service/src/main/docker/files/entrypoint.sh

```diff
@@ -120,5 +120,5 @@ fi
 
 cd /app
 exec java $JAVA_OPTS $JAVA_OPTS_OVERRIDE --add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED \
--Dlogback.debug=true -cp /app:/app/lib/* com.demandware.carbonj.service.engine.CarbonJServiceMain \
---spring.config.location=$CONFIG_LOCATIONS
+--add-opens java.base/java.util=ALL-UNNAMED -Dlogback.debug=true -cp /app:/app/lib/* com.demandware.carbonj.service.engine.CarbonJServiceMain \
```

Comment:
Any reason for this?

Reply:
Yes, apparently we are serializing objects from java.lang in CarbonJEvents. If we don't add this, CJ will start spitting out runtime exceptions..